tree_sitter/__init__.py

```diff
@@ -1,14 +1,22 @@
-from types import MethodType
-from ctypes import cdll, c_void_p
+import platform
+from ctypes import c_void_p
+from ctypes import cdll
 from ctypes.util import find_library
 from distutils.ccompiler import new_compiler
+from os import path
 from tempfile import TemporaryDirectory
-from tree_sitter_binding import Parser, Tree, language_field_id_for_name
-import os.path as path
-import platform
+
+from tree_sitter.binding import _language_field_id_for_name
+from tree_sitter.binding import Node
+from tree_sitter.binding import Parser
+from tree_sitter.binding import Tree
+from tree_sitter.binding import TreeCursor
```

Comment:
What is the expected use case for these exports? Is there some standard Python tooling that lets you see the docs of exported classes?

My only concern with this is that it makes it look like it you are allowed to instantiate these classes directly, like `tree = Tree()`. What will happen if people do this right now?

Reply:
> What is the expected use case for these exports? Is there some standard Python tooling that lets you see the docs of exported classes?

`pydoc` and `help` among others, see my other comment.

> But in reality, that is not supported. What will happen if people do this right now?

```
>>> tree_sitter.Tree()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: cannot create 'tree_sitter.Tree' instances